arcsi/api/show.py

```diff
@@ -85,28 +91,19 @@ def list_shows():
             show.cover_image_url = do.download(
                 show.archive_lahmastore_base_url, show.cover_image_url
             )
-    return many_show_details_basic_schema.dumps(shows)
+    return shows_schedule_schema.dumps(shows)
 
-@arcsi.route("/show/<id>", methods=["GET"])
-def view_show(id):
+
+@arcsi.route("/shows/archive", methods=["GET"])
```

Comment:
I misunderstood our communication on chats a bit, I thought you replaced `/show/<id>` with `/show/<slug>` this way however `/shows/archive` doesn't quite replace `/show/<id>`

A simple way to see could be to check the function call instead of the route. The one we would like to replace is `view_show(id)` which gives us the data of a single show from the db (with its episodes included in a hacky wrong way). The one you've added is called `list_shows_archive()` and has two main differences: it is defined without the `id` parameter and is querying all of Show.

I don't mean by this that your work was for nothing and we revert this, I think a lightweight list of all shows you created is very useful for a response to some listing requests